internal/errors/http/echo/default.go

```diff
@@ -0,0 +1,69 @@
+// package echo
+//
+// This package implements the default error handler to be
+// used for the api service to match the openapi specification.
+//
+// See: https://echo.labstack.com/docs/error-handling#custom-http-error-handler
+package echo
+
+import (
+	"errors"
+	"fmt"
+	"log/slog"
+	"net/http"
+	"strconv"
+
+	"github.com/labstack/echo/v4"
+	"github.com/openlyinc/pointy"
+	api_public "github.com/podengo-project/idmsvc-backend/internal/api/public"
+	app_context "github.com/podengo-project/idmsvc-backend/internal/infrastructure/context"
+)
+
+func DefaultErrorHandler(err error, c echo.Context) {
+	logger := app_context.LogFromCtx(c.Request().Context())
+	if c.Response().Committed {
+		logger.Warn("response already commited",
```

Comment:
Isn't it more a debug or trace level message? 

Btw this part and with `HandleError: true,` in log middleware config. The processing is done only once. But with the Warn statement, the same message is reported 3 times in log - in logger middleware, then the "encapsulating error" and then "response already committed".